### ISSUE TITLE
Hide / reorder expressions

### ISSUE THREAD

**Issue by stephanhuebsch** [SYtor/Kanji-Dojo#130]
It would be nice to have the option of hiding and/or reordering certain rare expressions of a kanji. 

For instance, the first expression of 会 is 会/え/gathering (which I've never seen before to be honest). I'd love to be able to hide this expression, or at least reorder the list, such that e.g. 会う or 会社 is at the top.

**Comment by syt0r** [comment]
I agree, there are probably a lot of ranking mistakes so I added an option to report expression, but ability to hide them instantly would be better for users, and I'll be able to add reports to understand what expressions are getting hidden too. It might get complicated to implement it though so for now I'll just lower rank of this expression and try to get back to this feature in future

**Comment by stephanhuebsch** [comment]
After giving it a bit more thought, the following would probably my ideal case scenario: 

1. hide all expressions regarding words I don't know 
2. present the list of all non-hidden expressions in random order

E.g., if I always see [あ]う and [かい]社 as the first expressions, I'll probably know that the kanji is 会. However, I won't be recalling that e.g. [かい]話 also needs 会, so it might be useful to opt-in to switch these expressions around in order to view all of them. 

Anyway, thanks for this amazing app, it really is a joy and pleasure to use!
